lib/routes/misskey/user-timeline.ts

```diff
@@ -3,13 +3,28 @@ import utils from './utils';
 import { config } from '@/config';
 import ConfigNotFoundError from '@/errors/types/config-not-found';
 import InvalidParameterError from '@/errors/types/invalid-parameter';
+import { fallback, queryToBoolean } from '@/utils/readable-social';
+import querystring from 'querystring';
 
 export const route: Route = {
-    path: '/users/notes/:username',
+    path: '/users/notes/:username/:routeParams?',
     categories: ['social-media', 'popular'],
     view: ViewType.SocialMedia,
     example: '/misskey/users/notes/support@misskey.io',
-    parameters: { username: 'misskey username format, like support@misskey.io' },
+    parameters: {
+        username: 'Misskey username in the format of username@instance.domain',
+        routeParams: `
+    | Key         | Description                        | Accepted Values | Default |
+    | ----------- | ---------------------------------- | --------------- | ------- |
+    | withRenotes | Include renotes in the timeline    | 0/1/true/false  | false   |
+    | mediaOnly   | Only return posts containing media | 0/1/true/false  | false   |
+
+    Note: \`withRenotes\` and \`mediaOnly\` are mutually exclusive and cannot both be set to true.
+
+    Examples:
+    • /misskey/users/notes/mttb2ccp@misskey.io/withRenotes=true
+    • /misskey/users/notes/mttb2ccp@misskey.io/mediaOnly=true`,
+    },
```

Comment:
```suggestion
| Key         | Description                        | Accepted Values | Default |
| ----------- | ---------------------------------- | --------------- | ------- |
| withRenotes | Include renotes in the timeline    | 0/1/true/false  | false   |
| mediaOnly   | Only return posts containing media | 0/1/true/false  | false   |

Note: \`withRenotes\` and \`mediaOnly\` are mutually exclusive and cannot both be set to true.

Examples:
- /misskey/users/notes/mttb2ccp@misskey.io/withRenotes=true
- /misskey/users/notes/mttb2ccp@misskey.io/mediaOnly=true`,
    },
```
Indent with either 0 or 2 spaces or the markdown content won't be rendered properly.

Reply:
👌